RocketChat/Rocket.Chat.ReactNative
# [NEW] Support new message parser

## Proposed changes


Adds support for the new message parser.

## Issue(s)	


## How to test or reproduce


In order to test this feature you need to check that you have the new message parser enabled from the server global preferences and in the app go to Profile > Preferences > Enable Message Parser

## Screenshots

## Types of changes




## Checklist



## Further comments


Some edge cases may fail because they're not yet part of the message parser (e.g. formatted links)
